packages/server/integrations/gitlab/GitLabServerManager.ts

```diff
@@ -1,22 +1,20 @@
 import {GraphQLResolveInfo} from 'graphql'
+import {RootSchema} from '../../graphql/rootSchema'
 import getProfile from '../../graphql/nestedSchema/GitLab/queries/getProfile.graphql'
-import {GitLabRequest} from '../../graphql/rootSchema'
-import {IntegrationProviderGitLabOAuth2} from '../../postgres/queries/getIntegrationProvidersByIds'
 import {GetProfileQuery} from '../../types/gitlabTypes'
 
 class GitLabServerManager {
-  readonly provider: IntegrationProviderGitLabOAuth2
   readonly accessToken: string
   readonly serverBaseUrl: string
 
-  constructor(accessToken: string, serverBaseUrl: string) {
+  constructor(accessToken: string) {
     this.accessToken = accessToken
-    this.serverBaseUrl = serverBaseUrl
+    this.serverBaseUrl = 'https://gitlab.com'
```

Comment:
`serverBaseUrl` will always be `gitlab.com` so we don't need to include this as an arg? Instead, I've added it here

Reply:
that's not true for self-hosted versions